openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/upsell_cta.txt

```diff
@@ -1,5 +1,4 @@
 {% load i18n %}
-
```

Comment:
FYI @edx/rapid-experiments-team  I removed extra lines from here and the includers of this file - to reduce the line breakage to 2 lines from 3 lines in the text-based emails.